src/lib/Validator/ValidationErrorsProcessor.php

```diff
@@ -51,7 +51,7 @@ public function processValidationErrors(array $validationErrors): void
             $violationBuilder->setParameters($message->values);
 
             if ($propertyPathGenerator !== null) {
-                $propertyPath = $propertyPathGenerator($i, $error->getTarget());
+                $propertyPath = $propertyPathGenerator($i, (string)$error->getTarget());
```

Comment:
Target is expected to be null, propertyPathGenerator needs to accept `?string` and handle it properly.

Reply:
Good, fixed in `FieldValueValidator::generatePropertyPath` instead